docs/how-to/deploy/deploy-spaces.md

```diff
@@ -0,0 +1,53 @@
+# Deploy on Juju spaces
+
+The Charmed Apache Kafka operator supports [Juju spaces](https://documentation.ubuntu.com/juju/latest/reference/space/index.html) to separate network traffic for:
+
+- **Internal communications**, including inter-broker and broker-to-controller communications.
+- **Client**, including traffic between broker and clients (producers and consumers).
+
+## Prerequisites
+
+* Charmed Apache Kafka 4
+* Configured network spaces
+  * See [Juju | How to manage network spaces](https://documentation.ubuntu.com/juju/latest/reference/juju-cli/list-of-juju-cli-commands/add-space/)
+
+## Deploy
+
+On application deployment, constraints are required to ensure the unit(s) have address(es) on the specified network space(s), and endpoint binding(s) for the space(s).
+
+For example, consider the output of `juju spaces` command is like below:
+
+```text
+Name      Space ID  Subnets
+alpha     0         10.148.97.0/24
+client    1         10.0.0.0/24
+peers     2         10.10.10.0/24
+```
+
+The space `alpha` is the default and cannot be removed. Two other spaces are configured for client and peer (internal) communications. To deploy Charmed Apache Kafka using the mentioned spaces:
+
+```bash
+juju deploy kafka --channel 4/edge \
+  --constraints spaces=client,peers \
+  --bind "cluster=peers kafka-client=clients"
+```
+
+[note type=caution]
```

Comment:
Todo: Fix admonition syntax. This is the BBcode syntax from Discourse times. We use the following instead in RTD:

````
```{caution}

```
````